modules/multi2multivec-jinaai/module.go

```diff
@@ -0,0 +1,137 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2025 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package modclip
```

Comment:
question: why is this package called "clip" if `Name = "multi2multivec-jinaai"`? Should it be called something with "m2mjinaai" to reflect the module name?

Reply:
will change this, thanks for spotting this!

Reply:
done